packages/dcos-log/extra/dcos-log-master.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=DC/OS Log service master: DC/OS logging service
+Description=DC/OS Log Master: exposes component, container, and task logs
```

Comment:
I clarified with @darkonie: 
- dcos-log master and agent both expose journald logs (host & component) internally
- dcos-log agent also exposes container (task) logs internally
- admin router does the proxying to expose both APIs externally